datasets/subjqa/README.md

```diff
@@ -0,0 +1,285 @@
+---
+annotations_creators:
+- expert-generated
+language_creators:
+- found
+languages:
+- en
+licenses:
+- unknown
+multilinguality:
+- monolingual
+size_categories:
+- 1K<n<10K
+source_datasets:
+- original
```

Comment:
From the licensing information we know that it's based on other datasets: Amazon Review Dataset from UCSD, The TripAdvisor Dataset from UIUC's Database and Information Systems Laboratory, The Yelp Dataset.

So I would add 
```suggestion
- original
- yelp_review_full
- other-amazon_reviews_ucsd
- other-tripadvisor_reviews
```
(we don't have the amazon and tripadvisor datasets in the library)

Reply:
good idea - i did not realise we could link to other datasets on the hub this way!